core/blockly.js

```diff
@@ -80,12 +80,6 @@ Object.defineProperty(Blockly, 'mainWorkspace', {
   }
 });
 
-/**
- * Currently selected block.
- * @type {?Blockly.ICopyable}
- */
-Blockly.selected = null;
```

Comment:
I believe we will be able to do get/set accessors (despite the current difficulties 🤞), and this probably should have a pair.

Reply:
Done!